Slim/Networking/Slimproto.pm

```diff
@@ -310,6 +316,10 @@ sub forget_disconnected_client {
 		$cprefs->set('positionAtDisconnect', $position);
```

Comment:
Looking into https://github.com/michaelherger/RadioParadise/issues/6 I believe we must check whether the currently playing track can seek: the protocol handlers have a `canSeek` method which. If that returns falsy, we should store a position of `0`.

Reply:
Ah - I see you have committed this in [#699](https://github.com/Logitech/slimserver/commit/304ba5885465ebf55a559553315bcf162770acf7).

That was done after I had drafted this PR.

I have now rebased this PR to accomodate your commit, all is as expected.